### ISSUE TITLE
[bug]: ./space/.env.example is missing

### ISSUE THREAD

**Issue by Hack4Life** [makeplane/plane#3103]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

When running the setup script it fails with the following error:
```
/home/plane# ./setup.sh
cp: cannot stat './space/.env.example': No such file or directory
```
Looks like the `.env.example` file is missing inside the `space` directory. However for `web` and `apiserver` the `.env.example` file is present.

### Steps to reproduce

1. `git clone --depth 1 -b master https://github.com/makeplane/plane.git && cd plane`
2. `./setup.sh`

### Browser

Other

### Version

Self-hosted

**Comment by pablohashescobar** [comment]
Thanks for pointing this out I have created a PR #3105 to fix this.

**Comment by shelkesays** [comment]
> ### Is there an existing issue for this?
> * [x]  I have searched the existing issues
> 
> ### Current behavior
> When running the setup script it fails with the following error:
> 
> ```
> /home/plane# ./setup.sh
> cp: cannot stat './space/.env.example': No such file or directory
> ```
> 
> Looks like the `.env.example` file is missing inside the `space` directory. However for `web` and `apiserver` the `.env.example` file is present.
> 
> ### Steps to reproduce
> 1. `git clone --depth 1 -b master https://github.com/makeplane/plane.git && cd plane`
> 2. `./setup.sh`
> 
> ### Browser
> Other
> 
> ### Version
> Self-hosted

I see the same problem. This is not allowing to launch of the space container. Is the `.env`  really required anymore for `space`? As for the PR shows the line copying `space/.env.example` to `space/.env` is removed.

**Comment by pablohashescobar** [comment]
Hi @Hack4Life, this error is fixed in the current release `0.14.1-dev`. You can clone the latest `master` or `preview` branch.
